front-end/src/pages/HomePage.tsx

```diff
@@ -57,6 +59,9 @@ export default function HomePage(props:RouteComponentProps) {
         <h3>Feed</h3>
         {postListElToDisplay}
       </Col>
+      <Col>
+        <Button onClick={() => createPost("/create_post")}>Create Post</Button>
```

Comment:
Nit: could we replace `createPost()` with `props.history.push('/create_post');`?

Reply:
Done